api-code-examples/nodejs/blobs-list-incomplete-blobs.rs

```diff
@@ -1,14 +0,0 @@
-import { Iroh } from '@number0/iroh'
```

Comment:
don't delete me, just fix my extension

Reply:
🫡

Reply:
erm, wait, file is already there. `.rs` was a duplicate